### ISSUE TITLE
Screenshots not taken on:error when tests run in selenium grid

### ISSUE THREAD

**Issue by christopher-hitchens** [QualityOps/wdio-timeline-reporter#49]
I am running my webdriverio tests in a remote selenium grid environment and using timeline-reporter for my results. When I run my tests locally timeline-reporter takes a screenshot after each test (not sure why) and on:error and embeds those screenshots into the report. When I run the same tests in the selenium grid timeline-reporter takes a screenshot after each test but not on:error. Using webdriverio v6 and wdio-timeline-reporter 5.1.1 and Mocha framework.

    services: [[TimelineService]],
...
`    reporters: ['spec',
        ['timeline', {
            outputDir: './target/results',
            fileName: 'timeline-reporter.html',
            embedImages: true,
            images: {
                quality: 100,
            },
            screenshotStrategy: 'on:error'
        }]
    ],`
...

**Comment by benzaremean** [comment]
I will have a look at this

**Comment by tanyatech303** [comment]
@benzaremean Same error with WebdriverIO. Any updates? Even locally does not work. Throwing "Cannot read property 'includes' of undefined"

**Comment by dagoud** [comment]
> @benzaremean Same error with WebdriverIO. Any updates? Even locally does not work. Throwing "Cannot read property 'includes' of undefined"

Yes I have found this too - when using automationProtocol: 'devtools' in my wdio.conf.js.

When removing this the screenshots are generated but ignores 'on:error'. So we get a screenshot for every test.

**Comment by eznxxy** [comment]
is this issue has any solution yet?
